dynamo/data_io.py

```diff
@@ -334,3 +352,63 @@ def export_rank_xlsx(adata, path="rank_info.xlsx", ext="excel", rank_prefix="ran
             if key[: len(rank_prefix)] == rank_prefix:
                 main_info("saving sheet: " + str(key))
                 adata.uns[key].to_excel(writer, sheet_name=str(key))
+
+
+def export_kmc(adata: AnnData) -> None:
+    """Save the parameters of kmc and delete the kmc object from anndata."""
+    kmc = adata.uns["kmc"]
+    adata.uns["kmc_params"] = {
+        "P": kmc.P,
+        "Idx": kmc.Idx,
+        "eignum": kmc.eignum,
+        "D": kmc.D,
+        "U": kmc.U,
+        "W": kmc.W,
+        "W_inv": kmc.W_inv,
+        "Kd": kmc.Kd,
+    }
+    adata.uns.pop("kmc")
```

Comment:
why not add this to the place where kmc is saved to adata.uns?

Reply:
In this way, we can keep a `kmc` object when we run the Dynamo analysis. If we need it, we just read it from the `adata.uns`.

`Umap` object uses a different saving strategy, which means umap parameters will be saved instead of the object itself. My idea is that it is possible to avoid creating a umap in the pipeline even if we run the umap dimension reduction. Unless the user wants to perform inverse transform in a specific analysis like `fate`, they don't need this Umap instance. While the creation of `KMC` is inevitable if the user enables the `kmc` method in `tl.cell_velocities`.  Since we have already created `KMC`, we can keep it for future usage until saving to h5ad.